### ISSUE TITLE
Dashes in hash.d documentation breaks Ddoc

### ISSUE THREAD

**Issue by zorael** [ikod/cachetools#1]
Manjaro/Arch x86_64, dmd v2.083.0. dlang-requests 1.0.3, cachetools 0.0.6.

Projects that have cachetools as a dependency cannot generate Ddoc docs.json, because of the unbalanced dashes at https://github.com/ikod/cachetools/blob/master/source/cachetools/hash.d#L7.

Andrew Pany at https://forum.dlang.org/post/hvkaitygjjkmythozcse@forum.dlang.org:
>The compiler (DMD) is triggered to generate a JSON file docs.json which contains technical information used by ddox in a later step. The error message you see is thrown by dmd.

```d
///
/// For classes (and structs with toHash method) we use v.toHash() to compute hash.
/// -------------------------------------------------------------------------------
/// toHash method CAN BE @nogc or not. HashMap 'nogc' properties is inherited from this method.
/// toHash method MUST BE @safe or @trusted, as all HashMap code alredy safe.
///
/// See also: https://dlang.org/spec/hash-map.html#using_classes_as_key 
/// and https://dlang.org/spec/hash-map.html#using_struct_as_key
///
```

```
$ dub build -b docs
Performing "docs" build using /usr/bin/dmd for x86_64.
cachetools 0.0.6: building configuration "library"...
../../../.dub/packages/cachetools-0.0.6/cachetools/source/cachetools/hash.d(14,6): Error: unmatched --- in DDoc comment
/usr/bin/dmd failed with exit code 1.
```

I guess the simplest solution would be to replace the dashes with equal signs.
```patch
diff --git a/source/cachetools/hash.d b/source/cachetools/hash.d
index cd1fcb5..9429a85 100644
--- a/source/cachetools/hash.d
+++ b/source/cachetools/hash.d
@@ -4,7 +4,7 @@ import std.traits;
///
/// For classes (and structs with toHash method) we use v.toHash() to compute hash.
-/// -------------------------------------------------------------------------------
+/// ===============================================================================
/// toHash method CAN BE @nogc or not. HashMap 'nogc' properties is inherited from this method.
/// toHash method MUST BE @safe or @trusted, as all HashMap code alredy safe.
///
```

**Comment by ikod** [comment]
Hello @zorael 

Sorry for that, will fix very soon.

**Comment by ikod** [comment]
fixed in v0.0.7 release

**Comment by zorael** [comment]
Thanks for the quick response!
